_posts/2020-06-28-openfaas-loki-into.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Time travelling through your OpenFaaS Function logs
+description: "In this post Lucas will introduce the openfaas-loki log provider and how to use it to get historic logs for your OpenFaaS functions."
+date: 2020-05-28
+image: /images/2020-openfaas-loki/2020-06-loki-logs-timey-wobbly.png
```

Comment:
This image is 2.91 MB and exceeds the soft limit in our writing guidelines - https://github.com/openfaas/openfaas.github.io#images

Please could you fix and force push that?